graphql/src/main/graphql/com/github/twitch4j/graphql/internal/fetchChannelPointRewards.graphql

```diff
@@ -0,0 +1,138 @@
+query fetchChannelPointRewards($login: String!) {
```

Comment:
this technically should be `fetchCommunityPointsSettings`, but don't think that's clear enough to end users that it would contain the custom reward list